common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java

```diff
@@ -257,31 +263,71 @@ private void addParam(RestParam param) {
     paramMap.put(param.getParamName(), param);
   }
 
+  public ProduceProcessor findProduceProcessor(String type, String serialView) {
```

Comment:
Maybe this code can be better. Because:


ProduceProcessor can be constructed at initialization, and will not change when excepting requests. So only `findProduceProcessor(String type)` can expose to the callers. `findProduceProcessor(String type, String serialView)` is not needed. And `ensureFindProduceProcessor` do not need to processing annotations for each request, but process once when initialization.

Reply:
done, I will init processor at **RestOperationMeta#createProduceProcessors** .